dh2000Sup/dh2000.proto

```diff
@@ -0,0 +1,21 @@
+Terminator = '\r';
+ReplyTimeout = 2000;
+
+getStatus {
+    out "\&STS!";
+    in "\&ACK!";
+    in "\&A%d,I%(\$1)d!";
+}
+
+openShutter {
+    out "\&OPENA!";
+    in "\&ACK!";
+
+}
+
+closeShutter {
+    out "\&CLOSEA!";
+    in "\&ACK!";
+
+}
```

Comment:
As per comment in DB make these two commands a single protocol entry using the enum format converter.